#Scenes/SceneScripts/TestingScene.gd

```diff
@@ -5,5 +5,25 @@ extends Node2D
 ## Now that the main scene is the main menu and we load the testing scene from there, we init to properly init the game state 
 
 func _ready() -> void:
+	_play_music()
 	SaveManager.execute_save()
 	PhaseManager.initialize_game()
+
+func _play_music() -> void:
+	var number_of_floors: float = MapManager.get_floors()
+	
+	# Offset by 1 since it's reading from an array
+	var current_player_floor: float = PlayerManager.player_position.y + 1
+	
+	var percentage_complete: float = (current_player_floor / number_of_floors) * 100
```

Comment:
you can use `MapManager.get_map_percent_with_player_position` instead to be consistent